content.html

```diff
@@ -0,0 +1,139 @@
+	<li>ordered list&#32;(<code>ol</code>) - level&#32;1
+		<ol>
+			<li>ordered list&#32;(<code>ol</code>) - level&#32;2</li>
+			<li>ordered list&#32;(<code>ol</code>) - level&#32;2
+				<ol>
+					<li>ordered list&#32;(<code>ol</code>) - level&#32;3</li>
+					<li>ordered list&#32;(<code>ol</code>) - level&#32;3</li>
+				</ol>
+			</li>
+		</ol>
+	</li>
+</ol>
+<table class="table table-striped">
+	<caption>Table caption</caption>
+	<thead>
+		<tr>
+			<th scope="col">Table header&#32;(<code>th</code>)</th>
+			<th scope="col">Table header&#32;(<code>th</code>)</th>
+			<th scope="col">Table header&#32;(<code>th</code>)</th>
+			<th scope="col">Table header&#32;(<code>th</code>)</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+		</tr>
+		<tr>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+		</tr>
+		<tr>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+			<td>Table data&#32;(<code>td</code>)</td>
+		</tr>
+	</tbody>
+</table>
+<form method="post" action="#" class="form-horizontal">
+	<div class="form-group">
+		<label for="data1" class="col-sm-3 control-label">Form&#32;<code>input</code></label>
+		<div class="col-sm-9">
+			<input type="text" id="data1" name="data1" class="form-control" />
+		</div>
+	</div>
+	<div class="form-group">
+		<label for="data2" class="col-sm-3 control-label">Form&#32;<code>textarea</code></label>
+		<div class="col-sm-9">
+			<textarea id="data2" rows="3" cols="15" name="data2" class="form-control"></textarea>
+		</div>
+	</div>
+	<div class="form-group">
+		<label for="data4" class="col-sm-3 control-label">Form&#32;<code>select</code></label>
+		<div class="col-sm-9">
+			<select name="data4" id="data4" class="form-control">
+				<option value="1">Option&#32;1</option>
+				<option value="2">Option&#32;2</option>
+				<option value="3">Option&#32;3</option>
+				<option value="4">Option&#32;4</option>
+			</select>
+		</div>
+	</div>
+	<div class="form-group">
+		<div class="col-sm-offset-3 col-sm-9">
+			<button type="submit" class="btn btn-primary">Submit</button>
+		</div>
+	</div>
+</form>
+<blockquote>
+	<p>"<code>blockquote</code>"</p>
+	<footer><code>footer</code> <cite><code>cite</code></cite></footer>
+</blockquote>
+
+<h2 id="cnt-wdth-lmtd">Section with limited width content</h2>
+<p>Add the CSS class name <code>.cnt-wdth-lmtd</code> to a sectioning element <code>&lt;section class="cnt-wdth-lmtd"&gt;...&lt;section&gt;</code> inside the main content of your page. More guidance are provided in the Content and IA specification.</p>
+<section class="cnt-wdth-lmtd">
```

Comment:
Apply "content limited width" on all pages/subpages. That should be the default.
+ you can remove this example of content limited width